tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Testing/NUnit/NUnitIntegration.cs

```diff
@@ -178,7 +172,7 @@ internal static void FinishScope(Scope scope, Exception ex)
                     scope.Span.SetTag(TestTags.Status, TestTags.StatusPass);
                     scope.Span.SetTag(TestTags.Message, ex.Message);
                 }
-                else if (exTypeName == "NUnit.Framework.IgnoreException" || exTypeName == "NUnit.Framework.InconclusiveException")
+                else if (exTypeName is "NUnit.Framework.IgnoreException" or "NUnit.Framework.InconclusiveException")
```

Comment:
I like these updates, but wondering if you did it for any reason other than "Rider told me to"? 🙂

Reply:
"Rider told me to"